cl/scrapers/management/commands/cl_scrape_opinions.py

```diff
@@ -209,7 +209,7 @@ def add_arguments(self, parser):
             help="Disable duplicate aborting.",
         )
 
-    def scrape_court(self, site, full_crawl=False):
+    def scrape_court(self, site, full_crawl=False, ocr_available=True):
```

Comment:
Missing a chance to add typing.

Reply:
I think I added typing here at first, but cut my losses when it didn't easily work.